docs/src/dplyr.md

```diff
@@ -0,0 +1,293 @@
+# DataFramesMeta.jl tutorial
+
+This is a port of the HarvardX series PH525x Genomics class tutorial on dplyr. View the original [here](https://genomicsclass.github.io/book/pages/dplyr_tutorial.html) and the source [here](https://github.com/genomicsclass/labs/blob/master/intro/dplyr_tutorial.Rmd).
+
+## What is DataFramesMeta.jl?
+
+DataFramesMeta.jl is a Julia package to transform and summarize tabular data. It provides a more convenient syntax to work with DataFrames from [DataFrames.jl](https://github.com/JuliaData/DataFrames.jl). For a deeper explanation of DataFramesMeta.jl, see the [documentation](https://github.com/JuliaData/DataFramesMeta.jl). 
+
+DataFramesMeta.jl is heavily inspired by R's `dplyr`. If you are familiar with `dplyr` this guide should get you up to speed with DataFramesMeta.jl. 
+
+## Why Is It Useful?
```

Comment:
capitalization of words in headings needs to be reviewed (not a native so I might miss some rules, but it seems that now it is inconsistent across headers)